concent_api/core/subtask_helpers.py

```diff
@@ -80,7 +89,7 @@ def update_timed_out_subtasks(
                 subtask             = subtask,
             )
         elif subtask.state == Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER.name:  # pylint: disable=no-member
-            locked_subtask = Subtask.objects.select_for_update().get(pk=subtask.pk)
+            locked_subtask = Subtask.objects.get(pk=subtask.pk)
```

Comment:
Locking in the query at the beginning of this function should cover this and below cases.

Reply:
Right, done. :+1:  for this